### DISCUSSION TITLE
Associate two events (SQS and S3) to the same function (Lambda)

### DISCUSSION THREAD

**Discussion by jhernandis** [brefphp/bref#1735] [General]
Hi,

We've one Lambda using the SqsHandler and works fine. But from AWS suggest us to use S3 events to trigger that Lambda, but we still need to launch it using an SQS message on other scenario / use case, so we are searching the way to not to duplicate that lambda but let trigger by S3 event and SQS message, both. Can we do that using bref?

Thanks.

**Comment by mnapoli** [comment]
You can, but that makes little sense in most scenarios TBH. Why not upload the same codebase in 2 Lambda functions?

**Reply by jhernandis** [reply]
Yes, that's it's the alternative we're evaluating. 

Thanks.
